src/components/ProjectionForm.js

```diff
@@ -43,7 +43,7 @@ function ProjectionForm() {
 
   return (
     <Form className={styles.form}>
-      {projectionOptions.forestType && (
+      {projectionMode === 'manual' && projectionOptions.forestType && (
```

Comment:
Why? This field should be visible but disabled in projectionMode manual.

Reply:
Its because the dropdown options remain always there even in map mode. Also when the error message is there.

Reply:
Let's maybe change this in another PR.

Reply:
done!